### ISSUE TITLE
[Request] Textbundle as default storage format?

### ISSUE THREAD

**Issue by magicnemo** [Zettlr/Zettlr#417]
[Feature-request]
Hi!

Would it be possible to add an option to (always) use "Text-bundle" format by default for newly created notes, i.e. not store images/attachments and text separately as default, but instead for every new created note in Zettlr also create corresponding text-bundle package?

(Thus avoiding exporting each note in textbundle format, in case you also have other apps that reads folders with textbundle files)

//
BR Anders

**Comment by nathanlesage** [comment]
Hey,

thanks for the issue! Unfortunately, this is not possible. There are several reasons to that:

1. Both TextBundle and TextPack have been designed specifically for inter-application sharing of notes and cross-device sharing. They were never intended to be used as a default storage container.
2. TextBundles are basically just folders within folders, so using these by default would inflate the amount of folders within your application unnecessarily; additionally, information will be stored redundantly, thereby consuming more space than with the current setup. Additionally, it would take a longer startup time, as more information needs to be processed, and the current File System Abstraction Layer already has a lot to do parsing 10,000+ files due to the lack of caching methods.
3. The "good" way to work with Markdown files is to store notes in one directory and use a single `img`, `assets`, or `images`-directory to collect **all** images being used across the notes in that directory. This means: If you need an image a second time, you can simply link it to both files. Additionally, you'll have more control over what is stored where. Zettlr already gives you the option to automatically store pasted images in a predefined "collection" folder.
4. Markdown is intended to be used as a format that is both machine-readable and human-readable; to add additional layers of information makes it harder to write programs to address these issues.

TL;DR: The default way is and will be to have single Markdown files with the images in a different folder. TextBundle and TextPack, however, can be used to share specific information with other users, devices, or applications.
